### DISCUSSION TITLE
Ios : Full-screen customise playback controls

### DISCUSSION THREAD

**Discussion by SivasubramanianE** [TheWidlarzGroup/react-native-video#4246] [Q&A] [answered]
### What are you trying to do?

In full-screen mode the player takes the RCT iOS default video player. I want the option to hide the playback options group like speed rate, subtitle, languages.

### Platforms

iOS

**Comment by KrzysztofMoch** [comment, marked as answer]
Hey @SivasubramanianE 
So it's currently impossible to do it, we are using default system player for fullscreen. Maybe there would be option to hide some elements but not sure if apple support it

**Reply by SivasubramanianE** [reply]
Ok thanks. I will check from my side
